celestia/peers.txt

```diff
@@ -1 +1 @@
-
+1b92a439555a19ee6edea85b184abb6873757dae@51.89.173.96:43656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

Reply:
> please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

https://github.com/celestiaorg/networks/pull/481 done

Reply:
cool will close this then